site/about.php

```diff
@@ -0,0 +1,38 @@
+<?php
+    require_once 'includes/app_config.php';
+?>
+
```

Comment:
Merge your require's into a single block of PHP code. This extra line is treated like a blank line of HTML, which is output before the doctype declaration in `header.php`